### PR TITLE
Feat/display chosen card

### PR DESCRIPTION
**Pull request by adrianammt**
Hello everyone. This is a late pre-weekend pr!🌈☀️
I am sorry it got a bit long in the process.
What is happening:
- If there are no classes saved a message component with a link to the create page is rendered
- When a class is created, it can now be clicked on and it opens a new page with the class id where a PlayClass component is rendered. This component receives the class information. For now it can be toggled favourite and it is saved in the object data. More functionality coming soon!
- Then some cleaning up and refactoring the code:
  - I was using two useStates to save the class information and using them in two different components. Now I have one state living in the parent App.js and it is passing the information top down.
  - All functions connected to the localStorage data are living in App.js (remove, save, toggleFavourite)
  - Functions are passed into the components in Main.js
I think that's the most important bit.

If someone is there tomorrow...I am looking forward to your reviews!!! Have a great weekend!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,12 +2,54 @@ import "./App.css";
...
+
+    const toggledClassIndex = listOfClasses.findIndex(
+      (savedClass) => savedClass.id === id
+    );
+    setListOfClasses([
+      ...listOfClasses.slice(0, toggledClassIndex),
+      toToggleClass[0],
+      ...listOfClasses.slice(toggledClassIndex + 1),
+    ]);
+  }
```
Comment (me-jterhorst) [inline src/App.js:42]: This one is really good :)

File: src/components/PlayClassCard/PlayClassCard.js [hunk 2]
```diff
@@ -35,9 +24,9 @@ export default function PlayClassCard(props) {
   }
 
   return (
-    <section className="PlayClass" {...rest}>
+    <section className="PlayClass">
       <IoHeart
-        className={isFavourite ? "FavIcon--inactive" : "FavIcon--active"}
+        className={isFavourite ? "FavIcon--active" : "FavIcon--inactive"}
```
Comment (me-jterhorst) [inline src/components/PlayClassCard/PlayClassCard.js:29]: I like what you did. But do you really need an inactive modifier or would it be easier to just have inactive as the default class and active as a modifier?

File: src/pages/PlayClass.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import PlayClassCard from "../components/PlayClassCard/PlayClassCard";
+
+export default function PlasClass({ data, toggleFavourite }) {
+  const { id } = useParams();
+
+  const classToPlay = data.filter((savedClass) => savedClass.id === id);
+
+  return (
+    <PlayClassCard
+      id={id}
```
Comment (me-jterhorst) [inline src/pages/PlayClass.js:11]: You might be better off calling it pageId or itemId to not confuse it with an html element id at some point.

**Comment by me-jterhorst** [comment]
I really like your changes so far. 
I noticed that when I was clicking on a selection / option element I couldn't see any entries unless i hovered over them, because they are having a default white color on a white background. 

Looking forward to see the next iteration! 👍

File: src/App.js (hunk 1 above)
Comment (adrianammt) [reply src/App.js:42]: Thanks! Can't take credit for it though, got some help here. ;)

File: src/components/PlayClassCard/PlayClassCard.js (hunk 2 above)
Comment (adrianammt) [reply src/components/PlayClassCard/PlayClassCard.js:29]: It is inactive per default. This is toggling back and forth.

**Pull request merged**